patches/minecraft/net/minecraft/world/WorldProvider.java.patch

```diff
@@ -152,6 +152,19 @@
 +        weatherRenderer = renderer;
 +    }
 +
++    /**
++     * Allows for manipulating the coloring of the lightmap texture.
++     * Will be called for each 16*16 combination of sky/block light values.
++     *
++     * @param sunBrightness Current sun brightness.
++     * @param skyLight Sky light brightness factor.
++     * @param blockLight Block light brightness factor.
++     * @param colors The color values that will be used: [r, g, b].
++     *
++     * @see net.minecraft.client.renderer.EntityRenderer#updateLightmap(float)
++     */
++    public void getLightmapColors(float sunBrightness, float skyLight, float blockLight, float[] colors) {}
```

Comment:
So as to be sure it's a nonnull array of size 3, to save a check where it's used.